.babelrc

```diff
@@ -1,3 +1,12 @@
 {
-  "presets": [ "@babel/preset-env" ]
-}
\ No newline at end of file
+  "presets": [
+    [
+      "@babel/preset-env", {
+        "targets": {
+          "ie": 11
```

Comment:
By default Babel will transpile es6 syntax but it will not automatically add polyfills for APIs such as `Array.prototype.filter` or `Object.assign`. Eventually we should look for a way to load all polyfills in the core so that they are available for all plugins without including them on each script